similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultMatchedPatientClusterView.java

```diff
@@ -137,6 +138,18 @@ private boolean convertEmpty(final int fromIndex, final int toIndex)
      */
     private JSONArray buildMatchesJSONArray(final int fromIndex, final int toIndex)
     {
+        // Sort by score
```

Comment:
this way we re-sort every time we request the next page. I think if we sort we should do it once when we initialise/instantiate this list (i.e. in constructor)

Reply:
As far as I understand we initialise this lidt each time we request a next page

Reply:
But sorting should happen in constructor, I'll relocate it.